Microsoft.Toolkit.Win32.Samples.WPF.WebView.HostingWinFormsWebView/app.manifest

```diff
@@ -0,0 +1,76 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@nmetulev @normesta @mcleanbyron Feedback from MVPs here as well: they're not crazy about requiring the manifest. It's a good practice to do starting with Windows 8.1. Beyond that, we really need what's on L55 with the changes in `app.config` to make sure the application gets all the high DPI support Windows 10 can offer